lib/api/api.go

```diff
@@ -446,6 +440,16 @@ func (s *service) CommitConfiguration(from, to config.Configuration) bool {
 	return true
 }
 
+func (s *service) addConfigHandler(handler http.Handler) http.Handler {
+	mux := http.NewServeMux()
+	configHandler := newConfigHandler(s.id, s.cfg)
+	mux.Handle(configBase, configHandler)
+	mux.Handle("/rest/system/config", configHandler)        // Legacy
+	mux.Handle("/rest/system/config/insync", configHandler) // Legacy
```

Comment:
"Legacy" seems a bit harsh since it's actively used by the GUI

Reply:
I changed the GUI to use the new endpoints and changed the labelling to "deprecated".